CHANGELOG.md

```diff
@@ -79,6 +79,7 @@ Note: You should always set `allow-deprecated-apis=false` to test for changes.
 - Fixed bogus "Bad commit_sig signature" which caused channel closures when reconnecting after updating fees under simultaneous bidirectional traffic.
 - Relative `--lightning_dir` is now working again.
 - Build: MacOS now builds again (missing pwritev).
+- Fixed `listtransactions` crash for unconfirmed channels or transactions.
```

Comment:
We recently started putting changelog entries in the commit messages that implement the changes (https://github.com/ElementsProject/lightning/pull/3238) this way we avoid the perennial conflicts on the `CHANGELOG.md` file :wink:

Reply:
thanks for the update :D